react/features/rtcstats/middleware.js

```diff
@@ -52,6 +53,39 @@ MiddlewareRegistry.register(store => next => action => {
         }
         break;
     }
+    case TRACK_ADDED: {
+        if (canSendRtcstatsData(state)) {
+            const jitsiTrack = action.track.jitsiTrack;
```

Comment:
I guess when adding tracks action.track cannot be undefined :)

Reply:
Right, but I am always worried about these "impossible" exceptions happening at random situations and places.. That said, we should be good here.

Reply:
We should be good, but I will make the code more defensive, just in case!